Source/Krypton Toolkit/Main/Krypton.Toolkit.Suite.Extended.Buttons/Controls Toolkit/Dialog Buttons/KryptonAbortDialogButton.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Krypton.Toolkit.Suite.Extended.Language.Model;
+using System;
+using System.ComponentModel;
+using System.Drawing;
+using System.Windows.Forms;
+
+namespace Krypton.Toolkit.Suite.Extended.Buttons
+{
+    [Designer(typeof(KryptonAbortDialogButtonDesigner)), Description(""), ToolboxBitmap(typeof(KryptonButton))]
+    public class KryptonAbortDialogButton : KryptonButton
+    {
+        private SelectedLanguage _language;
+
+        private KryptonForm _parent;
+
+        public SelectedLanguage ChosenLanguage { get => _language; set { _language = value; Invalidate(); } }
+
+        public KryptonForm ParentWindow { get => _parent; set { _parent = value; Invalidate(); } }
+
+        public KryptonAbortDialogButton()
+        {
+            ChosenLanguage = SelectedLanguage.ENGLISH;
+
+            DialogResult = DialogResult.Abort;
+
+            ParentChanged += KryptonAbortDialogButton_ParentChanged;
+
+            TextChanged += KryptonAbortDialogButton_TextChanged;
+        }
+
+        private void KryptonAbortDialogButton_TextChanged(object sender, EventArgs e)
+        {
+            if (Text == Name)
+            {
+                Text = "&Abort";
```

Comment:
If this is not to be Translated, then put an `@` before the first `"`

Reply:
This can be applied to all text's

Reply:
I need to do more work on this, i.e. if a language has been changed, then grab the equivalent from a string array.